pkgs/development/python-modules/zodb/default.nix

```diff
@@ -23,6 +24,14 @@ buildPythonPackage rec {
       sha256 = "20155942fa326e89ad8544225bafd74237af332ce9d7c7105a22318fe8269666";
     };
 
+    patches = [
+      # Compatibility with transaction v3.0
+      (fetchpatch {
+        url = "https://github.com/zopefoundation/ZODB/commit/0adcc6877f690186c97cc5da7e13788946d5e0df.patch";
+        sha256 = "122d6wkzgni704mifvgl1qzv353a6l6irlizhjkhmfqhpa05l3nh";
```

Comment:
fetchpatch will do some normalization
```suggestion
        sha256 = "1zmbgm7r36nj5w7icpinp61fm81svh2wk213pzr3l0jxzr9i5qi4";
```

```
$ nix-build ./nixpkgs/ -A python3Packages.zodb
these derivations will be built:
  /nix/store/48dd5z5xgvybramnpn87369ck3sc98vy-0adcc6877f690186c97cc5da7e13788946d5e0df.patch.drv
  /nix/store/n4pia2rqck73304yffpid8kw8j608ybc-python3.7-ZODB-5.5.1.drv
building '/nix/store/48dd5z5xgvybramnpn87369ck3sc98vy-0adcc6877f690186c97cc5da7e13788946d5e0df.patch.drv'...

trying https://github.com/zopefoundation/ZODB/commit/0adcc6877f690186c97cc5da7e13788946d5e0df.patch
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  7700    0  7700    0     0  45029      0 --:--:-- --:--:-- --:--:-- 45029
hash mismatch in fixed-output derivation '/nix/store/jzjpsk69j1nbwczjdpbra9gxcmx71zpx-0adcc6877f690186c97cc5da7e13788946d5e0df.patch':
  wanted: sha256:122d6wkzgni704mifvgl1qzv353a6l6irlizhjkhmfqhpa05l3nh
  got:    sha256:1zmbgm7r36nj5w7icpinp61fm81svh2wk213pzr3l0jxzr9i5qi4
cannot build derivation '/nix/store/n4pia2rqck73304yffpid8kw8j608ybc-python3.7-ZODB-5.5.1.drv': 1 dependencies couldn't be built
error: build of '/nix/store/n4pia2rqck73304yffpid8kw8j608ybc-python3.7-ZODB-5.5.1.drv' failed
```

Reply:
Ah, my bad, and using nix-prefetch-url hid the issue locally. Fixed.